config/drupal-phpunit-bootstrap-file.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+/**
+ * @file
+ *
+ * This fixes Drupal testing namespace autoloading and PHPUnit compatibility.
+ */
+
+use Rector\Core\Autoloading\BootstrapFilesIncluder;
+use Symplify\PackageBuilder\Parameter\ParameterProvider;
+
+assert($this instanceof BootstrapFilesIncluder);
+/** @phpstan-ignore-next-line */
+$parameterProvider = $this->parameterProvider;
+assert($parameterProvider instanceof ParameterProvider);
```

Comment:
@TomasVotruba how horrible of an atrocity am I committing here? We need the parameters in Rector to resolve the Drupal path for fixing some autoloading

Reply:
I see. Maybe it would be simpler to use `PhpFileLoader` from Symfony and load just `rector.php` via ContainerBuilder to get parameters.

Much cleaner than porting whole RectorFactory and building Rector again IMO

Reply:
This is inside of a `bootstrap_file` which gets required in the main scope, so it's not porting and building rector.

See: https://github.com/rectorphp/rector/blob/06f85e4a68acba9ea05d7894e79df5661663e452/src/Autoloading/BootstrapFilesIncluder.php#L38

I know PHPStan actually wraps the include in a static function to restrict the scope, but also passes a few things via `use ()`.

I didn't know if there was an easier way to have a bootstrap file / autoloader which was aware of the current parameters (without injecting globals)